tests/integration/targets/snap/tasks/main.yml

```diff
@@ -13,12 +13,10 @@
   block:
     - name: Include test
       ansible.builtin.include_tasks: test.yml
-    # TODO: Find better package to install from a channel - microk8s installation takes multiple minutes, and even removal takes one minute!
-    # - name: Include test_channel
-    #   ansible.builtin.include_tasks: test_channel.yml
-    # TODO: Find bettter package to download and install from sources - cider 1.6.0 takes over 35 seconds to install
-    # - name: Include test_dangerous
-    #   ansible.builtin.include_tasks: test_dangerous.yml
```

Comment:
cider is still pretty slow. Maybe it's faster to install snapcraft and build a small little snap that's unsigned?

In any case, could you comment out the `test_dangerous` again? Then we can merge this and already have quite an improvement to the current situation.

Reply:
done

Reply:
Thanks :)